src/explore-education-statistics-common/src/components/form/rhf/RHFFormNumberInput.tsx

```diff
@@ -0,0 +1,14 @@
+import FormBaseInput, {
+  FormBaseInputProps,
+} from '@common/components/form/FormBaseInput';
+import React from 'react';
+
+export interface RHFFormNumberInputProps extends FormBaseInputProps {
+  defaultValue?: number;
+  min?: number;
+  max?: number;
+}
+
+export default function RHFFormNumberInput(props: RHFFormNumberInputProps) {
```

Comment:
Given this'll only really get used by the form field component, probably worth just throwing this in the same file as `RHFFormFieldNumberInput` and not exporting it (same for the props)